suggestions/sqlite/sqlite.go

```diff
@@ -95,14 +97,63 @@ func (store *SqliteSuggestionsStore) Update(suggestion *suggestions.Suggestion)
 		sentInt = 1
 	}
 
-	_, err := store.db.Exec(
+	_, err := store.exec(
 		"UPDATE cfc_suggestions SET content_json=?, sent=?, message_id=? WHERE identifier=?",
 		contentJson, sentInt, suggestion.MessageID, suggestion.Identifier,
 	)
 
 	return err
 }
 
+func (store *SqliteSuggestionsStore) query(query string, args ...interface{}) (*sql.Rows, error) {
+	if store.LogQueries {
+		log.Infof("Sqlite query: \"%v\"  %v", query, args)
+	}
+	out, err := store.db.Query(query, args...)
+	if err != nil {
+		log.Info("Query errored: %v", err)
```

Comment:
Is this the correct usage of `Info` and `Infof` ?
Looks like they're both formatting strings so I didn't know

Reply:
yeah, that second one should be Infof ad well